cmd/skaffold/app/cmd/flags.go

```diff
@@ -277,6 +277,14 @@ var FlagRegistry = []Flag{
 		FlagAddMethod: "BoolVar",
 		DefinedOn:     []string{"dev", "run", "debug", "deploy", "render", "build", "delete", "diagnose"},
 	},
+	{
+		Name:          "no-labels",
```

Comment:
Please do not add negative boolean flags. They cause a lot of confusion, when someone says:

`--no-labels=false`

Instead, should this be `--add-skaffold-labels=false`?

Reply:
Will change to `--add-skaffold-labels=true` as default